akka-javasdk/src/main/java/akka/javasdk/annotations/Acl.java

```diff
@@ -37,27 +37,7 @@
    * annotated with their own @Acl definition.
    *
    */
-  DenyStatusCode denyCode() default DenyStatusCode.FORBIDDEN;
-
-  enum DenyStatusCode {
-    BAD_REQUEST(3),
-    FORBIDDEN(7),
-    NOT_FOUND(5),
-    AUTHENTICATION_REQUIRED(16),
-    CONFLICT(6),
-    INTERNAL_SERVER_ERROR(13),
-    SERVICE_UNAVAILABLE(14),
-    GATEWAY_TIMEOUT(4);
-
-
-
-    public final int value;
-    DenyStatusCode(int value) {
-      this.value = value;
-    }
-
-  }
-
+  int denyCode() default -1;
```

Comment:
Should we add something to the javadoc about the respective valid value ranges for HTTP and gRPC?

Reply:
Updated now.. see what you think.

had me questioning if we should limit the range of HTTP codes allowed in there.. currently we accept anything that's a valid http status code, be it an error or not.. but I think we can let the users decide.

Reply:
Sounds good to only limit valid HTTP codes, if users want to return 218 ("This is fine") or weirder it's up to them.